src/app/[locale]/specs/xrpc/en.mdx

```diff
@@ -187,6 +187,14 @@ Only HTTPS should be used over the open internet.
 
 Care should be taken with personally identifiable information in blobs, such as EXIF metadata. It is currently the *client's* responsibility to strip any sensitive EXIF metadata from blobs before uploading. It would be reasonable for a PDS to help prevent accidental metadata leakage as well; see future changes section below.
 
+## Redirects
+
+XRPC is built on top of HTTP. As such, everything that is not explicitly allowed or forbidden by the XRPC spec falls under the authority of HTTP.
+
+Unauthenticated "Query" (HTTP GET) XRPC endpoints, such as `getBlob`, support redirects. An explicit decision has not been made around XRPC support for redirects for "Procedure" (HTTP POST) endpoints and therefore should not be relied on.
```

Comment:
```suggestion
Unauthenticated XRPC Query endpoints (sent via HTTP GET requests), such as `getBlob`, do support redirects.

An explicit decision has not been made for XRPC Procedure endpoints (sent via HTTP POST requests) with regards to support for redirects and therefore should not be relied on.
```

Reworked the wording a little bit. Do we need the word "Unauthenticated" in the above? Can Authenticated XRPC Query's do redirects?